### ISSUE TITLE
Consider cookie cutter for generating basic layout

### ISSUE THREAD

**Issue by matentzn** [anitacaron/sssom-api#4]
https://github.com/tiangolo/full-stack-fastapi-postgresql

**Comment by matentzn** [comment]
Adopt the project layout from pheval:

https://github.com/monarch-initiative/pheval

Sooner rather than layer, in particular, adopt these files:

- Standard workflows: https://github.com/monarch-initiative/pheval/tree/main/.github/workflows
- And 
<img width="363" alt="image" src="https://user-images.githubusercontent.com/7070631/224471112-fd052ab1-de5c-48a2-98dd-e3ec14e48793.png">

**Comment by matentzn** [comment]
They are related - I changed the title; the point is to make a sane, standard repo structure with sane code style checking and integration testing, and a reasonably standard modular structure according to fastapi best practice

**Comment by anitacaron** [comment]
I was thinking of creating a cookie cutter for reuse, but that is different.

**Comment by matentzn** [comment]
You can, if you like! But this is less of a priority here then adding standardising this specific repo. My suggestion would be to first create a perfect layout here, and then, once we are happy with all of its aspects, transform it into a cookie cutter for monarch fastapi development?

**Comment by anitacaron** [comment]
Yes, entirely not the scope of this issue now and not previously.

**Comment by anitacaron** [comment]
Looking into the LICENSE in the pheval repo, it's missing to replace this part:

```
Copyright [yyyy] [name of copyright owner]
```

What does it need to be in the SSSOM API repo?

**Comment by matentzn** [comment]
Can you ask this question in the `#codestyle` channel in monarch org on slack?
